alcs-api/src/application/application-decision-meeting/application-decision-meeting.entity.ts

```diff
@@ -0,0 +1,26 @@
+import { AutoMap } from '@automapper/classes';
+import { Column, CreateDateColumn, Entity, ManyToOne } from 'typeorm';
+import { Base } from '../../common/entities/base.entity';
+import { Application } from '../application.entity';
+
+@Entity()
+export class ApplicationDecisionMeeting extends Base {
+  constructor(data?: Partial<ApplicationDecisionMeeting>) {
+    super();
+    if (data) {
+      Object.assign(this, data);
+    }
+  }
+
+  @AutoMap()
+  @CreateDateColumn({ type: 'timestamptz' })
+  date: Date;
+
+  @AutoMap()
+  @Column()
+  application_uuid: string;
+
+  @AutoMap()
+  @ManyToOne(() => Application)
```

Comment:
Its good to specify the reverse relationship as well since it is setup.
`@ManyToOne(() => Application, (app) => app.decisionMeetings)`

Reply:
based on the docs it is not a requirement

Reply:
huh I didnt know that, also looks like it doesnt do anything lol.